README.md

```diff
@@ -233,46 +233,47 @@ If you're implementing your authentication logic in a Fragment, you'll need to w
+	The Callback will have an object of AuthenticationResult which has accesstoken, date expired, and idtoken info. 
+	
+	Optional:  **acquireTokenSilent**
 
-Optional:  **acquireTokenSilent**
-
-You can call **acquireTokenSilent** to handle caching, and token refresh. It provides sync version as well. It accepts userid as paremeter.
+	You can call **acquireTokenSilent** to handle caching, and token refresh. It provides sync version as well. It accepts userid as paremeter.
  
-```java
-mContext.acquireTokenSilent(resource, clientid, userId, callback );
-```
+	```java
+	mContext.acquireTokenSilent(resource, clientid, userId, callback );
+	```
 11. Broker:
-  Microsoft Intune's Company portal app will provide the broker component. Adal will use the broker account, if there is one user account is created at this authenticator and Developer choose not to skip it. Developer can skip the broker user with:
-
-    ```java
-    AuthenticationSettings.Instance.setSkipBroker(true);
-    ```
- Developer needs to register special redirectUri for broker usage. RedirectUri is in the format of msauth://packagename/Base64UrlencodedSignature. You can get your redirecturi for your app using the script "brokerRedirectPrint.ps1" or use API call mContext.getBrokerRedirectUri. Signature is related to your signing certificates.
- 
- Current broker model is for one user. AuthenticationContext provides API method to get the broker user. 
-
- ```java
- String brokerAccount =  mContext.getBrokerUser();
- ```
- Broker user will be returned if account is valid. 
-
- Your app manifest should have permissions to use AccountManager accounts: http://developer.android.com/reference/android/accounts/AccountManager.html
-
- * GET_ACCOUNTS
- * USE_CREDENTIALS
- * MANAGE_ACCOUNTS
-
 
-Using this walkthrough, you should have what you need to successfully integrate with Azure Active Directory. For more examples of this working, viist the AzureADSamples/ repository on GitHub.
+	Microsoft Intune's Company portal App and Azure Authenticator App will provide the broker component. 
+	In order to acquire token via broker, the following requirements have to meet(Please check samples\userappwithbroker for authentication via broker):
+	* Starting version 1.1.14, developer has to explicitly specify set to use broker via:
+		```
+		AuthenticationSettings.Instance.setUseBroker(true);
+		```
+	* Developer needs to register special redirectUri for broker usage. RedirectUri is in the format of msauth://packagename/Base64UrlencodedSignature. You can get your redirecturi for your app using the script "brokerRedirectPrint.ps1" or use API call mContext.getBrokerRedirectUri. Signature is related to your signing certificates.
+	* If target version is lower than 23, calling app has to have the following permissions declared in manifest(http://developer.android.com/reference/android/accounts/AccountManager.html):
```

Comment:
We should really consider putting a bash script as well. I would assume most of the android developers are on some sort of unix platform.